bin/check-snmp.rb

```diff
@@ -46,6 +46,12 @@ class CheckSNMP < Sensu::Plugin::Check::CLI
          short: '-c critical',
          default: '20'
 
+  option :reverse,
+         short: '-r',
+         description: 'Revese the logic of the test for warning and critical thresholds',
```

Comment:
This is not an accurate description if I am not misunderstanding the code. I understand it as "Reverse the operator passed in". Am I missing something?